app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Api::V1::CommentsController < ApplicationController
+  load_and_authorize_resource
+
+  def index
+    @post = Post.find(params[:post_id])
+    @comments = @post.comments.all
+    render json: @comments
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.create(text: comment_parameters[:text], author_id: current_user.id)
+
+    if @comment.save
+      render json: @comment, status: 200
+    else
+      render json: @comment.errors, status: :bad_request, message: 'Failed to add comment'
+    end
+  end
```

Comment:
- [ ] Kindly fix this error when a new comment is created.

![image](https://user-images.githubusercontent.com/75126481/164944893-dab4757c-f3e2-4fed-87e6-43b21795bb6d.png)

Reply:
Working on it right now.

Reply:
Taking more time than required to fix.

Reply:
All fixed now